### DISCUSSION TITLE
希望3.0能增加短链接分享功能

### DISCUSSION THREAD

**Discussion by duansix** [AlistGo/alist#1735] [Ideas]
现状：目录有中文名 或者文件有中文名 分享出去的链接都是长链接 不是很好
想法：短链功能 方便分享

提示个小小的建议，望采纳

**Comment by MicroMatrixOrg** [comment]
我用CF 搭建了个短链接，自己用倒是可以。而且这个项目还有开放的API。不知道作者好不好加
https://github.com/crazypeace/Url-Shorten-Worker/
这是我的原链接https://bbs.micromatrix.org/d/519-%E3%81%91%E3%82%93%E7%A0%94%E3%81%91%E3%82%93%E3%81%91%E3%82%93%E3%83%95%E3%82%A7%E3%83%81%E9%83%A8%E3%81%AE%E6%B4%BB%E5%8B%95%E8%A8%98%E9%8C%B2200photos15
这是我转化之后的链接
https://url.micromatrix.org/nEKtHt
